src/coveralls.jl

```diff
@@ -185,4 +185,7 @@ module Coveralls
             println(String(r.body))
         end
     end
+
+    @deprecate submit_token submit_local
```

Comment:
This will need to be `Base.@deprecate`, assuming there isn't already a `using Base: @deprecate` in this module.

Reply:
OK... So why isn't that needed in src/codecovio.jl (where I took this from)?

Reply:
Whoa, weird... it looks like at some point `@deprecate` got exported from Base. Not sure when that happened. Never mind then.